components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -275,7 +267,8 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         @inject(ConfigCatClientFactory) private readonly configCatClientFactory: ConfigCatClientFactory,
 
-        @inject(Authorizer) protected readonly authorizer: Authorizer,
+        @inject(AuthRelationships) private readonly authRelationships: AuthRelationships,
+        @inject(AuthPermissions) private readonly authPermissions: AuthPermissions,
```

Comment:
```suggestion
        @inject(AuthPermissions) private readonly permission: AuthPermissions,
```

[nit] it would read slightly nicer to do `this.permission.onOrganization("org_id", OP, "user_id")`